cmd/tabulator/main.go

```diff
@@ -70,6 +71,7 @@ func gatherOptions() options {
 	flag.Var(&o.persistQueue, "persist-queue", "Load previous queue state from gs://path/to/queue-state.json and regularly save to it thereafter")
 	flag.StringVar(&o.creds, "gcp-service-account", "", "/path/to/gcp/creds (use local creds if empty)")
 	flag.BoolVar(&o.confirm, "confirm", false, "Upload data if set")
+	flag.BoolVar(&o.copyOnly, "copy-only", true, "Copies grid state to tab state with no transformation")
```

Comment:
Please default this to off until we roll it out to prod.

Reply:
It was defaulted "off", but it wasn't as clear I don't think. Switched the flag around